satpy/etc/readers/nwcsaf-pps_nc.yaml

```diff
@@ -120,25 +120,25 @@ datasets:
     name: ct
     file_type: nc_nwcsaf_ct
     coordinates: [lon, lat]
-    standard_name: cloudtype
+    standard_name: ct
```

Comment:
I think it was because I wanted to distinguish between the composite name and the dataset name, but probably I am not mastering the use/need of the `standard_name`? I can keep the `cloudtype` but not sure of the impact.

Reply:
If there is no CF standard name then this name only matters for matching enhancements at this point. If you update this standard_name here then an enhancement configuration should be added to match this standard_name (or have one for this specific product name `ct`). Or you can just let it find the default enhancement.

Reply:
I retverted back using `cloudtype`.